### PR TITLE
MAUI-5121 : Prepare Popup positioning UG for .NET MAUI popup

### PR DESCRIPTION
**Pull request by JayashreeRavishankar**
Prepare Popup positioning UG for .NET MAUI popup

### CODE REVIEW

File: MAUI/Popup/popup-positioning.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the center of the view.
+	sfPopup.IsOpen = true;
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:73]: highlight the line

File: MAUI/Popup/popup-positioning.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the center of the view.
+	sfPopup.Show();
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:101]: Highlight code

File: MAUI/Popup/popup-positioning.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at x-position 100 and y position 100.
+	 sfPopup.Show(50, 200);
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:133]: highlight the code

File: MAUI/Popup/popup-positioning.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+    <sfPopup:SfPopup x:Name="sfPopup">
+    </sfPopup:SfPopup>
+  </StackLayout>
+</ContentPage.Content>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the bottom of the button.
+	sfPopup.ShowRelativeToView(clickToShowPopup, PopupRelativePosition.AlignBottom, 0, 0);
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:164]: check the comment line alignment.
Highlight the code

File: MAUI/Popup/popup-positioning.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the center of the view.
+	sfPopup.Show();
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Popup with center positioning](Images/popup-positioning/maui-popup-with-center-positioning.png)
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:106]: ![Displaying a .NET MAUI Popup at the center of the screen.]
Include .NET MAUI in all image alt text

File: MAUI/Popup/popup-positioning.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+  <ContentPage.Content>       
+    <StackLayout>
+      <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" Margin="0,30,0,30"
+              HorizontalOptions="Center"
+              Clicked="ClickToShowPopup_Clicked" />
+      <Label x:Name="relativeView" Text="Showing Popup at relative position in MVVM" 
+             VerticalOptions="StartAndExpand" HorizontalOptions="Center" 
+             HorizontalTextAlignment="Center" FontFamily="Roboto" Padding="20"
+             LineBreakMode="WordWrap" FontSize="14" BackgroundColor="#6750A4" TextColor="White"/>
+      <sfPopup:SfPopup x:Name="sfPopup" IsOpen="{Binding DisplayPopup}"
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:225]: <sfPopup:SfPopup x:Name="sfPopup" 
                               IsOpen="{Binding DisplayPopup}".
                               
Modify like above and hightlight "IsOpen" code

File: MAUI/Popup/popup-positioning.md [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %} 
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the bottom of the label and with absolute relative position.
+  (this.BindingContext as ViewModel).DisplayPopup = true;
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:237]: highlight the code

File: MAUI/Popup/popup-positioning.md [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+{
+  // Shows SfPopup at the bottom of the button and with absolute relative position.
+	sfPopup.ShowRelativeToView(clickToShowPopup, PopupRelativePosition.AlignBottom, 50, 50);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Absolute positioning of Popup](Images/popup-positioning/maui-popup-absolute-relative-positioning.png)
+
+You can pass both negative and positive values as parameters to the SfPopup.ShowRelativeToView(View, RelativePosition, x-position, y-position). The popup will be positioned by considering the relative position as (0, 0) the center point. For example, if you have set the RelativePosition as RelativePosition.BottomRight and RelativeView as a button, bottom right corner of the button will be considered as the 0, 0 point and a negative x-position value will place the popup to the left of that point and a positive x-position value will place the popup to the right of that point. The same applies for y-position also.
```
Comment (PiruthivirajM) [inline MAUI/Popup/popup-positioning.md:200]: highlight the APIs in content

**Review by PiruthivirajM — COMMENTED** [review]
Address review comments

File: MAUI/Popup/popup-positioning.md [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+{
+  // Shows SfPopup at the bottom of the button and with absolute relative position.
+  sfPopup.ShowRelativeToView(clickToShowPopup, PopupRelativePosition.AlignBottom, 50, 50);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup at Absolute Relative position.](Images/popup-positioning/maui-popup-absolute-relative-positioning.png)
+
+You can pass both negative and positive values as parameters to the `SfPopup.ShowRelativeToView(View, RelativePosition, x-position, y-position)`. The popup will be positioned by considering the relative position as (0, 0) the center point. For example, if you have set the `RelativePosition` as `PopupRelativePosition.AlignBottomRight` and `RelativeView` as a button, bottom right corner of the button will be considered as the 0, 0 point and a negative x-position value will place the popup to the left of that point and a positive x-position value will place the popup to the right of that point. The same applies for y-position also.
```
Comment (M-A32) [inline MAUI/Popup/popup-positioning.md:200]: You can pass both negative and positive values as parameters to the `SfPopup.ShowRelativeToView(View, RelativePosition, x and y-position).` The popup will be positioned by considering the relative position as (0, 0) the center point. For example, if you have set the `RelativePosition` as `PopupRelativePosition.AlignBottomRight` and `RelativeView` as a button, bottom right corner of the button will be considered as the 0, 0 point and a negative x-position value will place the popup to the left of that point and a positive x-position value will place the popup to the right of that point. The same applies for y-position.

File: MAUI/Popup/popup-positioning.md [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+                       AbsoluteX="0" AbsoluteY="5">
+      </sfPopup:SfPopup>
+    </StackLayout>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4" %} 
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the bottom of the label and with absolute relative position.
```
Comment (M-A32) [inline MAUI/Popup/popup-positioning.md:237]: // Show the SfPopup at the bottom of the label and with absolute relative position.

**Review by M-A32 — CHANGES_REQUESTED** [review]

**Review by PiruthivirajM — APPROVED** [review]
Approved

**Pull request merged**